lib/common_dsl/generate/service_instance.rb

```diff
@@ -39,10 +39,21 @@ def self.add_service_dsl_files(service_instance, service_module_branch)
 
           dsl_version   = service_module_branch.dsl_version
           top_file_path = FileType::ServiceInstance::DSLFile::Top.canonical_path
+          check_for_assembly_wide(content_input)
           file_path__content_array = FileGenerator.generate_yaml_file_path__content_array(:service_instance, top_file_path, content_input, dsl_version)
           DirectoryGenerator.add_files(service_module_branch, file_path__content_array, donot_push_changes: true)
         end
         
+        def self.check_for_assembly_wide(content_input)
+          content_input[:asssembly][:workflows].each_pair do |k, v|
```

Comment:
This will fail if assembly part does not exist (which is edge case but can happen).